src/cpp/fastdds/subscriber/DataReaderImpl.hpp

```diff
@@ -399,6 +401,8 @@ class DataReaderImpl
     //! Requested incompatible QoS status
     RequestedIncompatibleQosStatus requested_incompatible_qos_status_;
 
+    SampleLostStatus sample_lost_status_;
```

Comment:
Document also the new members.

Reply:
Done on b09d55f7c1d1f39a1fb345126a46acf73e4c6845